package.json

```diff
@@ -19,6 +19,7 @@
     "@mui/icons-material": "^6.1.1",
     "@mui/material": "^6.1.1",
     "autoprefixer": "^10.4.20",
+    "axios": "^1.7.7",
```

Comment:
We don't need a separate module to handle http requests as of now. Since we're trying to have a static site let's keep the bundle size small.
We're trying to keep the size of static site export as light as possible.

Reply:
@tarun-29 can you please fix this, we can merge the PR.

Reply:
resolved @lucifercr07